templates/web/isleofwight/report/_council_sent_info.html

```diff
@@ -0,0 +1,17 @@
+[% SET duration_clause = problem.duration_string(c) %]
+[% IF duration_clause || problem.can_display_external_id %]
+    <p class="council_sent_info">
+    [%- IF problem.can_display_external_id %]
+        [%- IF duration_clause %]
+            [%- external_ref_clause = tprintf('Enquiry ref:&nbsp;%s', problem.external_id) %]
```

Comment:
If this is only line that changes from parent, might be good to factor out to separate template? But might be changed due to other changes, not sure.